### ISSUE TITLE
Test helpers to grand access for injectors in the Django url config and Celery app.

### ISSUE THREAD

**Issue by proofit404** [dry-python/dependencies#43]
#### Inject Mocker into Django URL config:

```python
from dependencies.contrib.django.test import override_route

class PaymentTest(TestCase):

    def test_submit(self):

        override_route(name='sumbit_payment_view', view_attr=Mock())
```

#### Inject Mocker into Celery app

```python
from dependencies.contrib.celery.test import override_task

class PaymentTest(TestCase):

    def test_submit(self):

        override_task(name='canvas.task.name', task_attr=Mock())
```

#### Blocked by:
- [ ] #9 
- [ ] #27

**Comment by proofit404** [comment]
Superseded by #192
